### PR TITLE
🚀  Add maven-publish plugin

### PR DESCRIPTION
**Pull request by hbernardi-luna**
## 📜 Description
Configure project `LbcTopAppBar` in order to deploy it on Sonatype and Maven central.
Additional documentation [here](https://www.notion.so/lunabeestudio/Lunabee-Android-Compose-bb08d8a58b4e46f1bce9394aa7b3b8ed)

I also removed all the Composable Preview from the project (as I think it is useless for the Demo App as it is already a preview, and for the library it will avoid to add unnecessary and unwanted dependencies).

## 💡 Motivation and Context
Publish the first library on Maven Central

## 💚 How did you test it?
Deploy the library here https://search.maven.org/artifact/studio.lunabee.compose/lbctopappbar
And on Snapshot repositories https://s01.oss.sonatype.org/content/repositories/snapshots/

Use implementation("studio.lunabee.compose:lbctopappbar:_") instead of     implementation(project(Modules.lbctopappbar)) in demo app.

If you want the release process, you will have to update the version. The last version on Maven Central is `0.9.1`.
You can use any versioning on a Snapshot repository, as long as version has a -SNAPSHOT suffix.

## 📝 Checklist
* [x] I compiled before submitting the PR
* [x] I reviewed the submitted code
* [x] I launched `./gradlew detekt`

## 🔮 Next steps
Add CI for automatization

### CODE REVIEW

File: buildSrc/src/main/kotlin/studio/lunabee/library/PublishingExt.kt [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+            url.set("https://github.com/LunabeeStudio/Lunabee_Compose_Android")
+        }
+
+        developers {
+            developer {
+                id.set("hbernardi-luna")
+                name.set("Hugo Bernardi")
+                email.set("hugo.bernardi@lunabee.com")
+            }
+        }
```
Comment (rlatapy-luna) [inline buildSrc/src/main/kotlin/studio/lunabee/library/PublishingExt.kt:148]: Move to gradle properties, or use publisher?

File: versions.properties [hunk 2]
```diff
@@ -1,5 +1,5 @@
 #
-# Copyright � 2022 Lunabee Studio
+# Copyright � 2022 Lunabee Studio
```
Comment (rlatapy-luna) [inline versions.properties:2]: 🤔

Comment (rlatapy-luna) [inline versions.properties:2]: 🤔

**Review by rlatapy-luna — CHANGES_REQUESTED** [review]

**Comment by hbernardi-luna** [comment]
@rlatapy-luna Configuration on Team City should be OK! I added a trigger to develop and master for the `Publish Library` job.

**Review by rlatapy-luna — APPROVED** [review]
StratisStraxGIF

**Pull request merged**